SECURITY.md

```diff
@@ -4,8 +4,9 @@
 
 | Version | Supported by us    | EOL                | Supported by distribution                                                         |
 | ------- | ------------------ | ------------------ | --------------------------------------------------------------------------------- |
-| 0.9.66  | :heavy_check_mark: |                    | :white_check_mark: Debian 11 **backports**, Debian 12 (testing/unstable)          |
-| 0.9.64  | :x:                |                    | :white_check_mark: Debian 10 **backports**, Debian 11, Ubuntu 21.04, Ubuntu 21.10 |
+| 0.9.68  | :heavy_check_mark: |                    |                                                                                   |
```

Comment:
Can you please move Debian 12 to 0.9.68? It's currently only in unstable, but will land in testing in ~5 days. And after that in Debian 11 backports.
Or should I update the file after it has migrated?

Reply:
I'd like to hold off on backports until it lands but for sure let's update for Sid!

Reply:
@Fred-Barclay It's now also in testing and bullseye-backports.